contrib/completion/bash/docker

```diff
@@ -1027,6 +1027,7 @@ _docker_run() {
 		--net
 		--pid
 		--publish -p
+		--read-bps-device
```

Comment:
Hi @tiborvass  --read-bps-device comes from cgroup file  blkio.throttle.read_bps_device. I think it's better to keep the same format with cgroup. If familiar with cgroup, it's easier for people to understand what they are going to set. But if you insisted, I think I'm OK to change it.